Documentation/intro.rst

```diff
@@ -0,0 +1,78 @@
+Introduction to Polycube
+========================
+
+Polycube brief
+--------------
+
+``Polycube`` is an **open source** software framework that provides **fast** and **lightweight** **network functions**. Already available services, called `cubes`, include `bridge`, `router`, `nat`, `load balancer`, `firewall`, `DDoS mitigator`, and more.
+
+Cube can be composed to build arbitrary **service chains** and provide custom network connectivity to **namespaces**, **containers**, **virtual machines**, and **physical hosts**.
+
+Cubes are extremely **efficient** because are based on the recent `BPF` and `XDP` Linux kernel technologies. In addition, cubes are easily **extensible** and **customizable**.
+
+Polycube networks can be controlled by a coherent, uniform and simple command line, named `polycubectl`, which interacts with a REST-based daemon, `polycubed`.
+
+Polycube provides also two proof-of-concept **standalone applications** based upon this framework.
```

Comment:
`proof-of-concept` -> `proofs-of-concept`

Reply:
Not sure it's correct. The plural refers to applications. Let's wait some native speaker to take care of this.